packages/gestalt/src/Switch.js

```diff
@@ -71,7 +71,7 @@ export default function Switch({
   const sliderStyles = classnames(
     styles.slider,
     switched ? styles.sliderRight : styles.sliderLeft,
-    switched && !disabled ? styles.sliderDark : styles.sliderLight,
+    !disabled ? styles.sliderDark : styles.sliderLight,
```

Comment:
Intentional change to help our default switch look active, and our disabled switch to look more disabled

Reply:
They still don't read as "disabled" enough to me, but this is an improvement! 😅 And that's beyond the scope of this PR